test/test_parser_ntriplesstar.py

```diff
@@ -0,0 +1,96 @@
+
+import pytest
+
+from pathlib import Path
+from shutil import copyfile
+from tempfile import TemporaryDirectory
+
+from rdflib.exceptions import ParserError
+
+from rdflib import Graph
+from rdflib.util import guess_format
+
+
+from rdflib.plugin import register
+from rdflib.parser import Parser
+
+register(
+    "ntstar",
+    Parser,
+    "rdflib.plugins.parsers.ntriples-star",
+    "NtriplesStarParser",
+)
+
+# tests should be past
```

Comment:
You should be using the N-Triples-star test manifest [[ref](https://github.com/w3c/rdf-star/blob/main/tests/nt/syntax/manifest.ttl)] for this together with our manifest processor [[ref](https://github.com/RDFLib/rdflib/blob/3a418218d6bcdb46f78342e14c024063e2f53e71/test/utils/dawg_manifest.py#L190-L203)]

For an example see:

https://github.com/RDFLib/rdflib/blob/3a418218d6bcdb46f78342e14c024063e2f53e71/test/test_w3c_spec/test_nt_w3c.py#L85-L96